### PR TITLE
Add Dutch translations

### PR DESCRIPTION
**Pull request by erikjanwestendorp**
Add Dutch translations

### CODE REVIEW

File: src/SeoToolkit.Umbraco.SiteAudit/wwwroot/SiteAudit/lang/nl-nl.xml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+﻿<?xml version="1.0" encoding="utf-8" standalone="yes"?>
+<language alias="nl">
+	<area alias="siteAudit">
+		<key alias="create">Audit Aanmaken</key>
+		<key alias="createAndStart">Aanmaken en strarten</key>
```
Comment (patrickdemooij9) [inline src/SeoToolkit.Umbraco.SiteAudit/wwwroot/SiteAudit/lang/nl-nl.xml]: Small typo here :)

File: src/SeoToolkit.Umbraco.ScriptManager/wwwroot/ScriptManager/lang/nl-nl.xml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+﻿<?xml version="1.0" encoding="utf-8" standalone="yes"?>
+<language alias="nl">
+	<area alias="siteManager">
+		<key alias="create">Script Aanmaken</key>
```
Comment (patrickdemooij9) [inline src/SeoToolkit.Umbraco.ScriptManager/wwwroot/ScriptManager/lang/nl-nl.xml]: And a small remark, I think "Aanmaken" shouldn't start with a capital letter, same for "Audit Aanmaken"

**Comment by patrickdemooij9** [comment]
Hi @erikjanwestendorp 
Super cool to see this! Such a small change, but it will really make it feel a lot better for Dutch users! I added two small comments, but the PR looks great otherwise. Thank you!

**Comment by erikjanwestendorp** [comment]
@patrickdemooij9 Thanks for your review, just updated te PR 😄

**Comment by patrickdemooij9** [comment]
And merged! Thank you for the PR!

**Pull request merged**